metadata-ingestion/cli-ingestion.md

```diff
@@ -1,66 +1,140 @@
+
+1. **server**: the location of the REST API exposed by your instance of DataHub
+2. **token**: a unique API key used to authenticate requests to your instance's REST API
+
+The token can be retrieved by logging in as admin. You can go to Settings page and generate a Personal Access Token with your desired expiration date.
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/saas/home-(1).png"/>
+</p>
+
+<p align="center">
+  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/saas/settings.png"/>
+</p>
+
+
+To configure your instance of DataHub as the destination for ingestion, set the "server" field of your recipe to point to your Acryl instance's domain suffixed by the path `/gms`, as shown below.
+A complete example of a DataHub recipe file, which reads from MySQL and writes into a DataHub instance:
+
 
 ```yaml
-# recipe.yml
+# example-recipe.yml
+
+# MySQL source configuration
 source:
-  type: <source_name>
+  type: mysql
   config:
-    option_1: <value>
-    ...
+    username: root
+    password: password
+    host_port: localhost:3306
 
+# Recipe sink configuration.
 sink:
-  type: <sink_type_name>
+  type: "datahub-rest"
   config:
-    ...
+    server: "https://<your domain name>.acryl.io/gms"
+    token: <Your API key>
 ```
+:::info Secure Your API Key
+Your API key is a signed JSON Web Token that is valid for 6 months from the date of issuance. Please keep this key secure & avoid sharing it. 
+If you are on Acryl DataHub and your key is compromised for any reason, please reach out to the Acryl team at support@acryl.io.:::
+:::
 
 For more information and examples on configuring recipes, please refer to [Recipes](recipe_overview.md).
 
-## Ingesting Metadata
-
-You can run ingestion using `datahub ingest` like below.
 
+## Ingesting Metadata
+The final step requires invoking the DataHub CLI to ingest metadata based on your recipe configuration file.
+To do so, simply run `datahub ingest` with a pointer to your YAML recipe file:
 ```shell
 datahub ingest -c <path/to/recipe.yml>
 ```
 
+## Scheduling Ingestion
+
+Ingestion can either be run in an ad-hoc manner by a system administrator or scheduled for repeated executions. Most commonly, ingestion will be run on a daily cadence.
+To schedule your ingestion job, we recommend using a job schedule like [Apache Airflow](https://airflow.apache.org/). In cases of simpler deployments, a CRON job scheduled on an always-up machine can also work.
+Note that each source system will require a separate recipe file. This allows you to schedule ingestion from different sources independently or together.
+
+:::note Real-time Ingestion
+Real-time ingestion setup is not recommended for an initial POC as it generally takes longer to configure and is prone to inevitable system errors.
+You can find more information on real-time ingestion [here](docs/lineage/airflow.md).
```

Comment:
let's remove this real-time ingestion thing

we can call out that some of our sources are pushed based and don't follow this recipe pattern